RELEASE-NOTES.md

```diff
@@ -0,0 +1,21 @@
+# Uberenv Software Release Notes
+
+Notes describing significant changes in each Uberenv release are documented
+in this file.
+
+The format of this file is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+The Uberenv project release numbers follow [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+### Added
+- Allow projects to force prefix on command line via new project option: `force_commandline_prefix`
```

Comment:
Could you please expand the "force prefix" part?

I.e. force the prefix for the library installation (or something a bit clearer).

Reply:
clarified.